### PR TITLE
Task #8320: Save Data for Edit Wallet Screen

### PR DESCRIPTION
**Pull request by tanbtd-1747**
### Issue Link :link:
https://edu-redmine.sun-asterisk.vn/issues/8320

### Implementation Details :construction:
Save data online when clicking Save or Delete Button.

### CODE REVIEW

**Comment by tanbtd-1747** [comment]
ready

**Review by tgdong2296 — APPROVED** [review]

File: Money*/Money*/Scenes/EditWallet/EditWalletViewController.swift [hunk 1]
```diff
@@ -64,6 +64,14 @@ final class EditWalletViewController: UIViewController {
         typeLabel.text = name
     }
     
+    private func updateData() {
+        wallet.ref?.updateData(wallet.dictionary)
```
Comment (tienpx3010) [inline Money*/Money*/Scenes/EditWallet/EditWalletViewController.swift]: Bắt sự kiện complete hay error đâu rồi (thichandonko)

File: Money*/Money*/Scenes/EditWallet/EditWalletViewController.swift [hunk 2]
```diff
@@ -64,6 +64,14 @@ final class EditWalletViewController: UIViewController {
...
+    }
+    
+    private func deleteData() {
+        wallet.ref?.delete()
```
Comment (tienpx3010) [inline Money*/Money*/Scenes/EditWallet/EditWalletViewController.swift]: Bắt sự kiện complete hay error đâu rồi (thichandonko)

**Review by minhduc2511 — APPROVED** [review]

**Comment by tanbtd-1747** [comment]
ready

**Review by tienpx3010 — APPROVED** [review]

**Pull request merged**